src/ml_gym/frontend/dashboard/src/routes/throughput_board.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { IOStatsType, JobStatusType } from "../app/datatypes"
+
+type ThroughputProps = {
+    ioStats: IOStatsType
+};
+
+
+const getThroughput = (ioStats: IOStatsType, measurementDuration: number) => {
+    const currentTime = new Date().getTime()
```

Comment:
Codacy has a fix for the issue: [Replace `··const·currentTime·=·new·Date().getTime()` with `const·currentTime·=·new·Date().getTime();`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
  const currentTime = new Date().getTime();
```